book/website/collaboration/ambassador-schemes/ambassador-schemes-establish.md

```diff
@@ -0,0 +1,85 @@
+
+(cl-ambassador-schemes-establish-running)=
+
+## Running the Scheme
+
+A useful start to the scheme is holding an onboarding meeting where you can bring together your ambassadors as a group to give information on how the scheme will work and let them get to know each other. 
+It may also be sensible to organise individual onboarding calls where you can set up participants on systems as needed and help brainstorm their work/activities for the scheme. 
+Additionallty you can provide an onboarding document or website, which participants can refer to throughout the scheme. 
+
+After this initial onboarding, plan how often your cohort will meet and how you will all communicate during the scheme - asynchronous tools like email or Slack are practical for this. 
+This will also feed into how you create a sense of community within your cohort. 
+At the end of your scheme, it may be nice to run a celebration or graduation type style event to celebrate everyone’s hard work.
+
+Once you’ve launched the scheme you can plan some communications around it to boost the profile of your ambassadors and to encourage the wider community to engage with them and their activities. 
+You may also want to make things like their project milestones and documentation open for good practice. 
+
+(cl-ambassador-schemes-establish-impact)=
+
+## Capturing the Impact of the Scheme
+
+Throughout the running of the scheme it is important to capture the impact. 
+The impact of the scheme may determine whether the scheme can be run again in terms of organisational or financial support. 
+This information is also important to record for reporting purposes for any organisation. 
+
+Recording impact could be done by keeping a spreadsheet of metrics such as how many events happened, how many people attended, how many resources were created and so on. 
+It could also involve qualitative metrics such as blog posts on projects and capturing testimonials from the ambassadors themselves or people in the community that have been involved in their activities.
+
+(cl-ambassador-schemes-establish-end-of-cohort)=
+
+## End of Ambassador Cohorts
+
+Whether you are continuing your ambassador scheme or only running one cohort, it is important to think about what needs to be done at the end of each cohort to close it down.
+
+As mentioned above, capturing metrics and testimonials for impact is important and will be part of this closing down process, but it is also important to get feedback. 
+This can be done verbally or through an electronic form.
+
+It is also important to archive the materials created for the scheme. 
+This will include the process documentation used to run the scheme and also outputs developed by the ambassadors. 
+It is best practice to make these resources openly available by depositing them in a FAIR manner using an [open repository such as Zenodo] {ref}`Steps for Making Research Objects Citable <cm-citable-steps>`.
```

Comment:
I'm not sure the "[open repository such as Zenodo]" is written/showing up as intended; it shows up exactly as written on the page